yarn-project/simulator/src/client/client_execution_context.ts

```diff
@@ -358,8 +360,15 @@ export class ClientExecutionContext extends ViewDataOracle {
    * @param encryptedNote - The encrypted data.
    * @param counter - The effects counter.
    */
-  public override emitEncryptedLog(encryptedData: Buffer, counter: number) {
-    const encryptedLog = new CountedLog(new EncryptedL2Log(encryptedData), counter);
+  public override emitEncryptedLog(
+    contractAddress: AztecAddress,
+    randomness: Fr,
+    encryptedData: Buffer,
+    counter: number,
+  ) {
+    // TODO(Miranda) use this.contractAddress? Pedersen hash inside the context or here?
+    const maskedContractAddress = pedersenHash([contractAddress, randomness], 0);
```

Comment:
I hashed here just to avoid adding more work for `private_context` - it only needs the `randomness`. Happy to change it though to avoid extra params for the oracle.